### ISSUE TITLE
[Accessibility] The spilt panel interactive control cannot be navigated to and/or operated with the keyboard alone.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227576]
- VS Code Version: 1.88.0
- OS Version: MacOS Sonoma (v14.6.1) 

**Issue**: When we contribute our extensions in left side bar with a split panel, navigating to it alone through keyboard isn't possible.

**[User Impact]**
Keyboard users will be prevented from accessing the functionality provided by these controls.

[Recommendation]
Ensure all interactive functionality is operable with the keyboard. The best way to accomplish this is by using appropriate native controls, which come with keyboard functionality built in.

If using a native control is not possible, the control must have tabindex="0" and have JavaScript event listeners that allow the control to be activated when Enter is pressed.

Screenshot of issue:
![Screenshot 2024-09-04 at 5 48 55 AM](https://github.com/user-attachments/assets/9ccd3e00-ebd2-4516-ad75-ba7eee722995)
![Screenshot 2024-09-04 at 5 49 50 AM](https://github.com/user-attachments/assets/15c80fe1-45e2-4f34-883c-8d0cd61fb650)

**Comment by meganrogge** [comment]
I am not sure what you mean by the "split panel interactive control"
